include/cli_startup.php

```diff
@@ -8,19 +8,16 @@
 // Everything we need to boot standalone 'background' processes
 
 function cli_startup() {
+	global $a;
 
-	global $a, $db;
-
-	if (is_null($a)) {
+	if (empty($a)) {
 		$a = new App(dirname(__DIR__));
 	}
 
-	if (is_null($db)) {
-		@include(".htconfig.php");
-		require_once("dba.php");
-		$db = new dba($db_host, $db_user, $db_pass, $db_data);
-		unset($db_host, $db_user, $db_pass, $db_data);
-	};
+	@include(".htconfig.php");
+	require_once("dba.php");
+	dba::connect($db_host, $db_user, $db_pass, $db_data);
```

Comment:
Can't this whole stuff be moved into a proper method? Encapsulation is the keyword.

Reply:
No. The function is also used in the install functionality, where the credentials have to be provided externally.

Reply:
Okay.